cmd/chainbridge/main.go

```diff
@@ -224,3 +230,12 @@ func run(ctx *cli.Context) error {
 
 	return nil
 }
+
+func getCurrentTag() string {
+
+	version := strings.TrimSpace(string(Version))
+
+	versionNumber := strings.Replace(version, "v", "", 1)
+
```

Comment:
Maybe we should add version logging on Chainbridge start then ?
Also then i do not see any need of removing replacing "v" from version, since as David mentioned, it could be a commit hash as well.

Reply:
the original code does not include 'v' , perhaps David / the team can explain why v is not used

Reply:
but this code will replace all "v"s in commit hash...

Reply:
This doesn't really serve any purpose at this point and will cause issues with commit hashes. 

Adding a log on startup would be good.

Reply:
code is now removed